charts/buffer/values.yaml

```diff
@@ -183,3 +183,67 @@ buffer:
 
 # cleanupAfterHandout -- Whether to publish message to Janitor after resource is handed out.
 cleanupAfterHandout: false
+
+liquibase-migration:
+  # In an env-specific file use something like the following to fully enable:
+  # ```
+  # liquibase-migration:   
+  #   defaults:
+  #     enabled: true
+  # ```
+  # See https://github.com/broadinstitute/terra-helm/blob/master/charts/liquibase-migration for more information
+
+  migrationJobs:
+  - name: buffer
+    # on filesystem, on classpath via app/resources/
+    migrationArgsConfigChangelog: "db/changelog.xml"
+    migrationDatabaseCredentials:
+      fromKubernetesSecret:
+        name: "buffer-postgres-db-creds"
+  - name: buffer-stairway
+    # inside stairway jar, on classpath via app/libs/*
+    migrationArgsConfigChangelog: "stairway/db/changelog.xml"
+    migrationDatabaseCredentials:
+      fromKubernetesSecret:
+        name: "buffer-stairway-db-creds"
```

Comment:
nit, Buffer and other apps are using` {{ .Values.name }} `not `buffer`
For example: `{{ .Values.name }}-postgres-db-creds`
https://github.com/broadinstitute/terra-helm/blob/master/charts/buffer/templates/secrets.yaml#L5

Reply:
This sort of extreme parameterization isn't something we can support well. The context in an application subchart doesn't point to the same `.Values` that we have access to at the root: the other `.Values.liquibase-migration` is visible to the sub-chart just as `.Values`, so things like `.Values.name` or `.Values.image` at the root aren't available.

We do have a fix: `.Values.global.*` is special, and _is_ available. We could move common "global for a chart" values there to give us more options. However, that's standardization we'd want to do across all charts, which is why we're effectively hardcoding the name for now. We are only enabling this for the live environments, and there the `.Values.name` always equals "buffer".

As a note, we are actually already having to account for this issue with this [very bad hack](https://github.com/broadinstitute/terra-helm/pull/513/files/16e833dcbd54eebf1c4b90577b1a9401d0d1feab#diff-f0039169139fa8741996dc6770e9d2579a1574c3c0b6c2fc2a36db470af63af0R212), and before we add `tpl` calls to render random strings (and encourage this pattern) we'd like to have the global thing sorted out